### PR TITLE
add draft post on outbreak analytics learning paths

### PR DESCRIPTION
**Pull request by avallecam**
Inspired by the ["Choose your learning path" section](https://education.rstudio.com/learn/) of the Rstudio education site, I collected and drafted this new entry for the website "Learn" tab.

This is currently a Draft PR, and a call to receive your feedback on the resources I included and what others I can add to this list.

Minor comment: Links are just pasted, but will be embedded as URL links in further edits.

My questions:
- Should this be a repository instead? This list of resources can have a periodic review process over time.
- Should we collapse all three knowledge domains into only three groups of beginner, intermediate, and expert? To give a clear message for Outbreak analytics learning path.
- Should we separate each on different sites to improve navigation? I think this should be doable by adding more qmd files inside the folder, isn't it?

Let me know your edit suggestions and questions regarding the content. 

> For edit suggestions, you can go to the "Files changed" tab (the fourth in the top row above this text box) to read the content as plain text and add your comments by clicking on the [+] on each line. Let me know if you prefer a different reading and editing format.

### CODE REVIEW

**Comment by avallecam** [comment]
Current look

Screenshot 2023-08-16 at 19-03-00 Epiverse-TRACE developer space - Learning paths

**Comment by avallecam** [comment]
> My questions:
> 
>     * Should we collapse all three knowledge domains into only three groups beginner, intermediate, and expert? To give a clear message for Outbreak analytics learning path.

I will check if the content supports this arrangement.

**Comment by avallecam** [comment]
I will evaluate to rename the modules using the epi training kit terminology ([spanish](https://epiverse-trace.github.io/epi-training-kit/), [english](https://www.flipsnack.com/breakfast2go/etk.html))

File: learn/paths/index.qmd [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+- **Extend your knowledge.** Watch Spanish LatinR Interactive <https://www.learnr4free.com/es/index.html> Videos <https://www.youtube.com/c/LatinR/playlists> Rladies Interactive <https://yabellini.shinyapps.io/RLadiesLesson/> Videos <https://www.youtube.com/@RLadiesGlobal/videos> 
+- **Publish your work with Quarto** <https://quarto.org/docs/get-started/>. R is excellent to analyse data and generate visualizations. If you want to show your work with narrative text in reports in PDF, MS Word, HTML, or presentations in MS PowerPoint, follow step 2 of the get started tutorial. If you prefer a video introduction before a dive into the tutorials, watch Mine Çetinkaya-Rundel video to author a Quarto document and create a website <https://youtu.be/_f3latmOhew> 
+:::
+
+::: {.callout-caution collapse="true"}
+### Experts
+
+- **Build your own R packages** <https://r-pkgs.org/>. Read the learning of the Epiverse-TRACE doing this as an organization <https://epiverse-trace.github.io/blueprints/> and follow good practices <https://devguide.ropensci.org/index.html> 
+- **Use Research compendium** for your data analysis project <https://frbcesab.github.io/rcompendium/>
```
Comment (avallecam) [inline learn/paths/index.qmd:52]:
```suggestion
- **Use Research compendium** for your data analysis project <https://frbcesab.github.io/rcompendium/> 
- **Use targets** for reproducible workflows <https://carpentries-incubator.github.io/targets-workshop/>
```

**Comment by avallecam** [comment]
evaluate to organize this in a tabset panel https://quarto.org/docs/interactive/layout.html#tabset-panel

**Comment by avallecam** [comment]
ai tools that provide learning paths
- https://skillai.io/
- https://explorer.globe.engineer/

**Comment by chartgerink** [comment]
Any updates on whether this PR will still be worked on? If not we can close it (please respond within next two weeks).

**Comment by avallecam** [comment]
> Any updates on whether this PR will still be worked on? If not we can close it (please respond within next two weeks).

Yes, it is on plans. To retake on June.

**Comment by chartgerink** [comment]
Thanks @avallecam - I now see this was still a draft so pardon if I came across as impatient! 😅 Sometimes I'm not sure whether PRs/issues are there because they have gone stale or for a reason.

**Comment by avallecam** [comment]
closing this because this will be the task of collaborators. thanks for the follow-up!

**Pull request closed without merging**